static/sass/_patterns_social.scss

```diff
@@ -0,0 +1,44 @@
+@mixin p-social {
+  .p-social__heading {
+    color: $color-mid-dark;
+    font-size: .875rem;
+    text-transform: uppercase;
+  }
+
+  .p-social__heading + * {
+    margin-top: .5rem;
+  }
+
+  .p-social__icon--twitter,
+  .p-social__icon--google,
+  .p-social__icon--facebook,
+  .p-social__icon--youtube {
```

Comment:
Layout of social icons can be done with `p-list--inline` pattern, so styles are mostly icons.

In theory they could be added to vanilla icons, but social icons differ a bit because they are links with hover state, so they are not just static icons like the others.

Reply:
Could you call this block `.p-social__icon` and then, inside each of the modified versions, add `@extends .p-social__icon`?

Reply:
Maybe you should also move the `background-image` bit into the modifiers, as it doesn't make much sense without the `background-position`